### ISSUE TITLE
Burkholderia JDBC rollback error with gmb3.0.1-build-2

### ISSUE THREAD

**Issue by kdahlquist** [lmu-bioinformatics/xmlpipedb#29]
I started an import/export cycle with the Burkholderia cenocepacia source files using gmb3.0.1-build-2. The UniProt XML imported successfully, but I got an error when attempting to import the GO obo-xml.  The error message and console screenshot are below.  The source files I used have been pushed to the "readme" branch.  I won't have a chance to run another export today, but I'll see if I can try over the weekend, otherwise it will be Monday.  I'm planning on doing Bordetella next.
![jdbc_rollback_error_20150115](https://cloud.githubusercontent.com/assets/6463867/12363479/deb285b4-bb7d-11e5-8497-557f5c5b87e2.jpg)
![jdbc_rollback_error_console_20150115](https://cloud.githubusercontent.com/assets/6463867/12363481/e07cb036-bb7d-11e5-95ad-c9983919e636.jpg)

**Comment by kdahlquist** [comment]
A Bordetella export finished, so this issue and #28 are possibly restricted to the Shewanella and Burkholderia custom profiles.  I will try Shigella next later today.

**Comment by dondi** [comment]
A full import-export cycle with the source files committed to the `readme` branch completed successfully for me. Although the specific error messages of "JDBC rollback failed/socket write error" are different from the one in issue #28, they do still fit the category of a cascading error resulting from a broken connection.

I will try another round with both species to see if the respective errors are encountered.

**Comment by kdahlquist** [comment]
I'll try another import/export cycle with Burkholderia (this issue) and Shewanella (#28) and let you know what happens.

**Comment by dondi** [comment]
I did another set of exports tonight and both completed successfully again, for both species.

**Comment by kdahlquist** [comment]
OK, good to know.  I won't have a chance to do this myself until tomorrow; I'm fully booked all day today.

**Comment by kdahlquist** [comment]
I just started an import/export cycle for Burkholderia without closing GenMAPP Builder from the previous Shewanella export.  I imported the UniProt XML, I started the import of the GO OBO-XML, and got an out of memory error.
![go_import_out-of-memory_20160121](https://cloud.githubusercontent.com/assets/6463867/12493613/a3c6deac-c03a-11e5-9375-62064029d552.jpg)
I am going to close GenMAPP Builder and also restart my machine and try again.
